LogMonitor/src/LogMonitor/ProcessMonitor.cpp

```diff
@@ -235,9 +235,21 @@ size_t bufferCopyAndSanitize(char* dst, char* src)
 ///
 size_t formatProcessLog(char* chBuf)
 {
-    // {"Source":"Process","LogEntry":{"Logline":"<chBuf>"},"SchemaVersion":"1.0.0"}
-    const char* prefix = "{\"Source\":\"Process\",\"LogEntry\":{\"Logline\":\"";
-    const char* suffix = "\"},\"SchemaVersion\":\"1.0.0\"}\n";
+    LoggerSettings settings;
+    const char* prefix;
+    const char* suffix;
+    if (Utility::CompareWStrings(settings.LogFormat, L"JSON"))
+    {
+        // {"Source":"Process","LogEntry":{"Logline":"<chBuf>"},"SchemaVersion":"1.0.0"}
+        prefix = "{\"Source\":\"Process\",\"LogEntry\":{\"Logline\":\"";
+        suffix = "\"},\"SchemaVersion\":\"1.0.0\"}\n";
+    }
+    else {
+        // <Source>Process</Source><LogEntry><Logline><chBuf>Z</Logline></LogEntry>
+        prefix = "<Source>Process</Source><LogEntry><Logline>";

```

Comment:
nit: same here `<LogEntry>` shoudl go at the beginning, before `<Source>`; as our intermediate schema for now.